index.html

```diff
@@ -1,56 +1,55 @@
-<!DOCTYPE html/>
```

Comment:
I do not think that this file needs to be removed or modified unless there is a hotfix.